SPECS/dbus/dbus.spec

```diff
@@ -1,14 +1,16 @@
 %{!?_versioneddocdir: %global _versioneddocdir %{_docdir}/%{name}-%{version}}
 Summary:        DBus for systemd
 Name:           dbus
-Version:        1.15.2
-Release:        4%{?dist}
+Version:        1.15.8
```

Comment:
I know this isn't part of the PR, but do you know why we're on 1.15.x at all? Per https://www.freedesktop.org/wiki/Software/dbus/, 1.15.x is a *development* branch and not an LTS / stable release branch for the dbus project.

In particular, I see this:

>The current development branch is [dbus 1.15.x](https://gitlab.freedesktop.org/dbus/dbus/blob/HEAD/NEWS), which will lead to a 1.16.x stable branch in future. The development branch does not come with security support, and if it requires security fixes, they will often be released as part of an ordinary development release that also contains potentially destabilising changes. Please do not include a development branch of dbus in your OS distribution unless you can commit to tracking that branch and the stable branch that follows it, for example continuing to track dbus 1.15.x and then switching to 1.16.x when it becomes available.